commands/blmpop.md

```diff
@@ -0,0 +1,33 @@
+`BLMPOP` is the blocking variant of `LMPOP`.
+
+When lists contains elements, this command behaves exactly like `LMPOP`.
+When used inside a `MULTI`/`EXEC` block, this command behaves exactly like `LMPOP`.
+When lists are empty, Redis will block the connection until another client
+pushes to it or until `timeout` (a double value specifying the maximum number
+of seconds to block) is reached.
```

Comment:
our new style guides (see readme) ask that we don't split sentences to multiple lines.
this is intended to make diffs smaller and code review easier.
i know it is confusing since most of the existing docs are written differently.
```suggestion
When lists are empty, Redis will block the connection until another client pushes to it or until `timeout` (a double value specifying the maximum number of seconds to block) is reached.
```

Reply:
i did take a look with the new guides and the existing code also confusing me...
anyway, i realize this now, and I will deal with it this way in the future